### ISSUE TITLE
[Question] Flux.Dev and Flux.Schnell

### ISSUE THREAD

**Issue by gafda** [NexaAI/nexa-sdk#75]
When are you planning on supporting the Flux.Dev and Flux.Schnell models?

Is there a roadmap of sorts?

**Comment by zhiyuan8** [comment]
Yes, this is on our roadmap and we expect to support it this week, including Flux.Dev, Flux.Schnell, SD3. Will keep you updated for the next release!

**Comment by zhiyuan8** [comment]
@gafda In our latest release, we have included flux
https://github.com/NexaAI/nexa-sdk/releases/tag/v0.0.8.3

**Comment by zhiyuan8** [comment]
Flux on CPU is support, GPU in progress
https://github.com/NexaAI/nexa-sdk/commit/80e4577db59c4b43d5c0d41b76a08e7a53a50379

**Comment by xsxszab** [comment]
Flux (including dev and schnell) are now supported on CUDA backends in the main branch
